### PR TITLE
Update webhooks docs

### PR DESCRIPTION
**Pull request by komalali**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/docs/pulumi-cloud/webhooks.md [hunk 1]
```diff
@@ -51,130 +45,85 @@ The Webhooks management page is on the Stack or Organization Settings tab.
 
 To create an organization webhook:
 
-1. Navigate to **Settings** > **Integrations**.
-1. Select **Create webhook**.
-1. Provide a _Display Name_, _Payload URL_, and optionally a _Secret_.
+1. Navigate to **Settings** > **Webhooks**.
+2. Select **Create webhook**.
+3. Select between a slack-formatted webhook or regular Pulumi webhooks.
```
Comment (susanev) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:50]: regular here means non-slack right?
if they use the other form for slack will it work?

**Review by susanev — APPROVED** [review]
we should figure out another docs solution for the rest api docs sometime this year.

thx for all the updates!!

File: themes/default/content/docs/pulumi-cloud/webhooks.md (hunk 1 above)
Comment (komalali) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:50]: > regular here means non-slack right?
yeah I should normalize on just using "generic" - will fix. 

>if they use the other form for slack will it work?
what is the other form you are referring to?

Comment (susanev) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:50]: Screenshot 2023-05-22 at 12 56 43 PM
 this one

File: themes/default/content/docs/pulumi-cloud/webhooks.md [hunk 2]
```diff
@@ -51,130 +45,85 @@ The Webhooks management page is on the Stack or Organization Settings tab.
...
+and optionally choosing which events you want delivered using [event filters](#event-filtering).
 
-expected_signature = base64.b64encode(hash.digest())
-```
+You can either create your own Slack app (or use an existing one you may already have installed in your workspace), or
+follow the link below to quickly get started with a pre-defined Slack app manifest.
 
-{{% /choosable %}}
-{{% choosable language go %}}
+[Slack App Manifest](https://api.slack.com/apps?new_app=1&manifest_yaml=display_information%3A%0A%20%20name%3A%20pulumi-slack-notifications%0A%20%20description%3A%20Funnel%20Pulumi%20webhooks%20to%20Slack%0A%20%20background_color%3A%20%22%238a3391%22%0Afeatures%3A%0A%20%20bot_user%3A%0A%20%20%20%20display_name%3A%20pulumi-slack-notifications%0A%20%20%20%20always_online%3A%20false%0Aoauth_config%3A%0A%20%20scopes%3A%0A%20%20%20%20bot%3A%0A%20%20%20%20%20%20-%20incoming-webhook%0Asettings%3A%0A%20%20org_deploy_enabled%3A%20false%0A%20%20socket_mode_enabled%3A%20false%0A%20%20token_rotation_enabled%3A%20false)
```
Comment (komalali) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:107]: @susanev I would like this link to be like a button or something. Any thoughts on how I might achieve this?

Comment (susanev) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:107]: @komalali im actively trying to remove html from our markdown. we can instead use a shortcode for buttons so we can keep things consistent.

for now though you should be able to grab similar html from the get started page with buttons.
https://github.com/pulumi/pulumi-hugo/blob/master/themes/default/content/docs/get-started/_index.md

pls use an o utline (secondary) button and not a filled-in (primary) button.
does that work for now?
@anita-trimbur and i can return to this in a few weeks and improve it!

Comment (komalali) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:107]: sounds good thanks!

File: themes/default/content/docs/pulumi-cloud/webhooks.md (hunk 1 above)
Comment (komalali) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:50]: this is the form that is being updated that these docs are referring to

Comment (komalali) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:50]: I think I'll need to update some screenshots.

Comment (susanev) [reply themes/default/content/docs/pulumi-cloud/webhooks.md]: oh sry this was from the cloud; ignore me if you've already got this covered

**Review by sean1588 — APPROVED** [review]

**Pull request merged**